tasks/tests/unit/test_notify_task.py

```diff
@@ -235,15 +238,26 @@ def test_determine_decoration_type_from_pull_attempt_activation(
             enriched_pull.database_pull.repository.owner.ownerid,
             pr_author.ownerid,
         )
-        assert mocked_send_task.call_count == 1
-        mocked_send_task.assert_called_with(
+        assert mocked_send_task.call_count == 2
+
+        new_user_activation_call = mocked_send_task.call_args_list[0]
+        account_user_activation_call = mocked_send_task.call_args_list[1]
+        assert new_user_activation_call == call(
```

Comment:
similar to the difference between `send_task` and `apply_async`, what is the difference between these two asserts? would it make sense to use the same patterns for both?

Reply:
The asserts deal with the different ways that `send_task` and `apply_async` the mocked `send_task`. Both asserts just check the call args (but the async call adds more stuff, which is why it's more complex).